.github/workflows/release.yml

```diff
@@ -0,0 +1,26 @@
+name: Publish package to the Maven Central Repository
+on:
+  push:
+    branches:
+      - dev
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Maven Central Repository
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+          server-id: ossrh
+          server-username: MAVEN_USERNAME
+          server-password: MAVEN_PASSWORD
+
+      - name: Release Maven package
+        uses: samuelmeuli/action-maven-publish@v1
+        with:
+          gpg_private_key: ${{ secrets.gpg_private_key }}
```

Comment:
Should be changed to uppercase:
secrets.GPG_PASSPHRASE
secrets.GPG_PRIVATE_KEY
secrets.NEXUS_PASSWORD
secrets.NEXUS_USERNAME

Reply:
fixed